app/src/main/java/de/danoeh/antennapod/fragment/preferences/NetworkPreferencesFragment.java

```diff
@@ -56,7 +58,13 @@ private void setupNetworkScreen() {
         });
         findPreference(UserPreferences.PREF_UPDATE_INTERVAL)
                 .setOnPreferenceClickListener(preference -> {
-                    new FeedRefreshIntervalDialog(getContext()).show();
+                    try {
+                        new FeedRefreshIntervalDialog(getContext()).show();
+                    } catch (InflateException e) {
+                        Toast.makeText(getContext(), "Unfortunately, Samsung messed up a system component"
```

Comment:
Can we be sure that it's happening only with Samsung? Can we make it brand-neutral?

```suggestion
                        Toast.makeText(getContext(), "Unfortunately, your phone manufacturer messed up standard Android functionality"
```

Reply:
I'm pretty sure it is only Samsung. Anyway, I found another workaround that might silently ignore the error and still display the time picker dialog.